MoneyDataType/KnownCurrencyTable.cs

```diff
@@ -1,49 +1,53 @@
+using Money.Abstractions;
+using System;
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
-using Money.Abstractions;
 
-namespace Money
+namespace Money;
+
+internal static class KnownCurrencyTable
 {
-    internal static class KnownCurrencyTable
-    {
-        private static readonly object Obj = new object();
+    private static readonly object _obj = new();
```

Comment:
Use a real name, e.g.
```suggestion
    private static readonly object _lockObject = new();
```